src/i_video.c

```diff
@@ -662,20 +655,23 @@ static void I_ToggleFullScreen(void)
     }
 }
 
-// [FG] the fullscreen variable gets toggled once by the menu code, so we
-// toggle it back here, it is then toggled again in I_ToggleFullScreen()
-
-void I_ToggleToggleFullScreen(void)
+void I_ToggleExclusiveFullScreen(void)
```

Comment:
Is this still a toggle?

Reply:
Yes, it toggles exclusive fullscreen if regular fullscreen is on. On Windows, if we simply switch to windowed mode from exclusive fullscreen, incorrect SDL resize messages and other issues occur.

Reply:
Understood, thanks!